fly.toml

```
@@ -0,0 +1,29 @@
+# fly.toml app configuration file generated for google-scrapper-ruby on 2023-05-18T11:25:54+07:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "google-scrapper-ruby"
+primary_region = "sin"
+console_command = "/rails/bin/rails console"
+
+[http_service]
+  internal_port = 3000
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+  min_machines_running = 0
+
+[checks]
+  [checks.status]
+    port = 3000
+    type = "http"
+    interval = "10s"
+    timeout = "2s"
+    grace_period = "5s"
+    method = "GET"
+    path = "/health_check"
+    protocol = "http"
+
+[env]
+    DEPLOYED_HOST = "google-scrapper-ruby.fly.dev"
```

It must be 2 spaces indent, no? 🙈 
```suggestion
  DEPLOYED_HOST = "google-scrapper-ruby.fly.dev"
```